libs/blocks/chart/chart.js

```diff
@@ -137,6 +146,8 @@ const init = async (el) => {
     chart.style.height = '600px';
   }
 
+  if (isNullish(window.IntersectionObserver)) return;
```

Comment:
Even if IntersectionObserver isn't available, we should probably still return the charts. Seems like you can put the conditional logic in the loadscript.then. Fail gracefully and all that

Reply:
nit: `isNullish` isn't needed
```suggestion
  if (!window.IntersectionObserver) return;
```

Reply:
@meganthecoder I moved this into loadScript.then and set the chart options if IntersectionObserver isn't available.